Tests/Resources/Fixtures/loader_config/valid3.yml

```diff
@@ -1,6 +1,10 @@
 stdClass:
-    uri_schema: /forum/{category}/{post_name}
+    definitions:
+        _default_:
+            uri_schema: /forum/{category}/{post_name}
 
 Symfony\Cmf\Component\RoutingAuto\Tests\Resources\Fixtures\ParentClass:
-    uri_schema: /forum/{category}
+    definitions:
+        _default_:
```

Comment:
do we really want people to use `_default_`? I would then rename it to `_default` or `default`